src/EnergyPlus/main.cc

```diff
@@ -63,6 +63,8 @@ using EnergyPlus::CommandLineInterface::ProcessArgs;
 int
 main( int argc, const char * argv[] )
 {
+	// the following line is only needed when debugging issues related to NaN in Visual Studio. See https://github.com/NREL/EnergyPlus/wiki/Debugging-Tips
+	// unsigned int fp_control_state = _controlfp( _EM_INEXACT, _MCW_EM );
```

Comment:
But if we were going to have it as an option, the /fp:strict should be added to CMAKE_CXX_FLAGS CMAKE_C_FLAGS which corresponds to C/C++ -> Code Generation -> Floating Pint Model in VS.